src/bionics.cpp

```diff
@@ -1075,10 +1075,19 @@ static bool attempt_recharge( Character &p, bionic &bio, units::energy &amount,
 void Character::process_bionic( int b )
 {
     bionic &bio = ( *my_bionics )[b];
-    if( calendar::once_every( 1_hours ) && !bio.id->fuel_opts.empty() &&
-        bio.has_flag( "AUTO_START_ON" ) && power_level <= 0_mJ ) {
-        g->u.activate_bionic( b );
+
+    if( !bio.id->fuel_opts.empty() && bio.has_var( "AUTO_START" ) ) {
+        const std::vector<itype_id> &fuel_available = get_fuel_available( bio.id );
+        const float start_threshold = std::stoi( bio.get_var( "AUTO_START" ) ) / 100.0;
+        if( !fuel_available.empty() && get_power_level() <= start_threshold * get_max_power_level() ) {
+            g->u.activate_bionic( b );
+        } else if( calendar::once_every( 1_hours ) ) {
```

Comment:
It looks like this will spam your log every hour that you have charge above the threshold you've set.
```suggestion
        } else if( get_power_level() <= start_threshold * get_max_power_level() && calendar::once_every( 1_hours ) ) {
```